eosdac-material/README.md

```diff
@@ -30,8 +36,23 @@ npm install
 ```
 cp src/statics/config.jungle.json src/statics/config.json
 ```
+*Please note that you may need to change the parameters of the jungle testnet config file periodically.*
+
 **6. Run dev server with material theme**
 ```
 quasar dev
 ```
 [Quasar Docs](https://quasar-framework.org/guide/index.html)
+
+## What to Change
+Since eosDAC is making this open source software for the use of creating other DACs, DACs will need to know what to change in order to quickly edit the software for their needs. The main repositories you will want to change for your DAC's branding, messaging, and assets are:
+- `src/statics`
+- `src/i18n`
+- `src/statics/config.json`
+
+`statics` is the main repository where you will want to change the files to better coincide with your DAC. Replace the files and retain the file names so you do not have to go through the other areas of the code to update the file names. You will also need to update the `i18n` directory for proper messaging and translations for the DAC's written content. The `config.json` file aligns with the proper deployed smart contract and you will need to change this file and the templates towards your DAC's smart contract.
+
+The main `package.json` file and other odd ball files may need to be changed by on the whole the `statics` and `i18n` folders and where you will want to make your changes.
```

Comment:
This sentence is a bit confusing to me. We may get rid of package.json all together in favor of yarn and I don't see why people would be modifying it anyway. Also, the view files should probably be mentioned in terms of people wanting to change the design of the interface, not just the language strings.